src/index.scss

```diff
@@ -107,6 +107,7 @@ h3 {
 
 .modalButton {
   &--back {
+    background: none;
```

Comment:
What does this fix?

Reply:
Removes background color from the filtering buttons on the top of the screenshots in Chrome; appears to be using the Chrome default button bg color